content/02.intro.md

```diff
@@ -19,7 +19,7 @@ In addition, the lack of established standards or concise recommendations for th
 biological questions further limits newcomers from using state-of-the-art deep learning in their research.  
 
 To make deep learning more accessible to biological researchers, we solicited input from a community of researchers with varied biological and deep learning interests.
-These individuals collaboratively contributed to this manuscript's writing using the GitHub version control platform [@url:https://github.com/Benjamin-Lee/deep-rules] and the Manubot manuscript generation toolset [@url:https://greenelab.github.io/meta-review/].
+These individuals collaboratively contributed to this manuscript's writing using the GitHub version control platform [@url:https://github.com/Benjamin-Lee/deep-rules] and the Manubot manuscript generation toolset [@url:https://greenelab.github.io/meta-review/; @10.1371/journal.pcbi.1007128 ].
```

Comment:
```suggestion
These individuals collaboratively contributed to this manuscript's writing using the GitHub version control platform [@url:https://github.com/Benjamin-Lee/deep-rules] and the Manubot manuscript generation toolset [@doi:10.1371/journal.pcbi.1007128].
```
I don't think we need to cite both the Manubot and journal version of the same paper.  How about using the DOI only?

Reply:
Good catch, forgot those were the same.